### DISCUSSION TITLE
Is there a way to enter passthrough mode on an input element?

### DISCUSSION THREAD

**Discussion by amano-kenji** [qutebrowser/qutebrowser#7242] [Q&A]
I want to choose an input element via hints and directly enter passthrough mode without going through insert mode. How can I do that?

**Comment by toofar** [comment]
I don't think you can unfortunately.

**Reply by The-Compiler** [reply]
Yup! @amano-kenji can you elaborate a bit on your use-case?

**Reply by amano-kenji** [reply]
I want to map enter key in insert mode to `leave-mode`. I often want to reserve insert mode for entering search queries.
If I enter passthrough mode directly on a textarea box, I can press enter as many times as I want.
